Detections/MultipleDataSources/BariumIPIOC112020.yaml

```diff
@@ -87,7 +87,7 @@ query:  |
   | where  DestinationIPAddress has_any (IPList)  
   | extend timestamp = TimeGenerated, IPCustomEntity = DestinationIPAddress, HostCustomEntity = Host 
   ), 
-  (imDns (response_has_any=IPList)
+  (imDns (response_has_ipv4=IPList)
```

Comment:
Thanks for noticing an notifying the error. since we are passing a dynamic list (IPList) I believe the fix should be: response_has_any_prefix.

Reply:
Fixed it.